lib/OrdinaryDiffEqSSPRK/test/ode_ssprk_tests.jl

```diff
@@ -491,3 +491,26 @@ end
 sol = solve(test_problem_ssp_long, alg, dt = OrdinaryDiffEqSSPRK.ssp_coefficient(alg),
     dense = false)
 @test all(sol.u .>= 0)
+
+@testset "VectorOfArray/StructArray compatibility" begin
+    using RecursiveArrayTools, StaticArrays, StructArrays
```

Comment:
I think you have to add StructArays.jl to the `Project.toml` in `lib/OrdinaryDiffEqSSPRK`.